posts/2019/10/uarray-attempting-to-move-the-ecosystem-forward.md

```diff
@@ -0,0 +1,72 @@
+<!--
+.. title: uarray: Attempting to move the ecosystem forward
+.. slug: uarray-attempting-to-move-the-ecosystem-forward
+.. date: 2019-10-25 04:55:54 UTC-05:00
+.. author: Hameer Abbasi
+.. tags: 
+.. category: 
+.. link: 
+.. description: 
+.. type: text
+-->
+
+There comes a time in every project where most technological hurdles have been surpassed, and its adoption is a social problem. I believe `uarray` and `unumpy` had reached such a state, a month ago.
+
+I then proceeded, along with [Ralf Gommers](https://github.com/rgommers) and [Peter Bell](https://github.com/peterbell10) to write [NumPy Enhancement Proposal 31](https://numpy.org/neps/nep-0031-uarray.html) Or NEP-31. This generated a lot of excellent feedback on the structure and the nuances of the proposal, which you can read both on the [pull request](https://numpy.org/neps/nep-0031-uarray.html) and on the [mailing list discussion](https://mail.python.org/pipermail/numpy-discussion/2019-September/079961.html), which led to a lot of restructuring in the contents and the structure of the NEP, but very little in the actual proposal. I take full responsibility for this: I have a bad tendency to assume everyone knows what I'm thinking. Thankfully, I'm not alone in this: It's a [known psychological phenomenon](https://en.wikipedia.org/wiki/Curse_of_knowledge).
+
+<!-- TEASER_END -->
+
+Of course, social problems can take a long time to resolve one way or another, regardless of the proponents. And I consider this a good thing: It's better to be stuck with an API decision that may bite you a few years down the line, especially with a project with API compatibility guarantees and number of dependents as NumPy.
```

Comment:
`It's` --> `it's`. 

"It's better to be stuck with an API decision" --> missing a "not"?